static/src/javascripts/types/global.d.ts

```diff
@@ -196,6 +196,26 @@ type AdBlockers = {
 	onDetect: function[];
 };
 
+type TagCounts = Record<string, number>;
```

Comment:
these new types are duplicated from elsewhere, because the build breaks if this file imports them

Reply:
Is this saying we can't import anything into `global`? That does seem odd?

Reply:
That does seem to be the case - typechecking breaks for every use of `window.guardian` in the codebase if I add an import to `global.d.ts`. I cannot understand why

Reply:
Could we maybe leave a note here saying these are duplicated so a new dev gets to this piece of code. Might try see why this is the case in another PR.